tests/test_wave_05.py

```diff
@@ -42,7 +42,8 @@ def test_get_goal(client, one_goal):
     assert response_body == {
         "goal": {
             "id": 1,
-            "title": "Build a habit of going outside daily"
+            "title": "Build a habit of going outside daily",
+            "tasks": []
```

Comment:
👀 We shouldn't as `tasks` here. The get one goal endpoint intentionally doesn't return the associated tasks when fetching the main Goal details. The called needs to opt in to receiving this data by using the `/goals/id/tasks` endpoint. Please revert this change.